### DISCUSSION TITLE
Student identity authentication

### DISCUSSION THREAD

**Discussion by xcdh-233** [community/community#49150] [GitHub Education]
My student ID is always rejected，there was no reply to the work order。How do I solve these problems？

**Comment by coderstar1234** [comment]
The change your id . The new one of id .

**Reply by xcdh-233** [reply]
I tried to send different proofs, but they were useless

**Comment by capozzz** [comment]
TB买

**Reply by xcdh-233** [reply]
想过 但是算了

**Reply by capozzz** [reply]
我刚buy了下.50.挺贵的

**Reply by xcdh-233** [reply]
我花了 30 RMB，开了学信网的英文证明，but毫无luan用
早知道直接某宝了
